WcaOnRails/app/controllers/translations_controller.rb

```diff
@@ -87,6 +87,30 @@ def update
   }.freeze
   # rubocop:enable Style/NumericLiterals
 
+  def self.translators_to_roles
```

Comment:
This should be feasible with `map` and `flat_map`:

```ruby
return VERIFIED_TRANSLATORS_BY_LOCALE.flat_map do |locale, user_ids|
  user_ids.map do |user_id|
    {
      ...role data...
    }
  end
end
```

Reply:
Done, this is nice.